indra/sources/tees/parse_tees.py

```diff
@@ -85,38 +85,39 @@ def parse_a1(a1_filename):
     """
     entities = {}
 
-    with open(a1_filename, 'r') as a1_file:
-        for line in a1_file:
-            tokens = line.rstrip().split('\t')
-            assert(len(tokens) == 3)
-
-            identifier = tokens[0]
-            entity_info = tokens[1]
-            entity_name = tokens[2]
-
-            info_tokens = entity_info.split()
-            assert(len(info_tokens) == 3)
-            entity_type = info_tokens[0]
-            first_offset = int(info_tokens[1])
-            second_offset = int(info_tokens[2])
-            offsets = (first_offset, second_offset)
-
-            entities[identifier] = TEESEntity(
-                    identifier,
-                    entity_type,
-                    entity_name,
-                    offsets)
+    for line in a1_text.split('\n'):
+        if len(line) == 0:
+            continue
+        tokens = line.rstrip().split('\t')
+        assert(len(tokens) == 3)
```

Comment:
Raise an exception (either a built-in one or a custom exception class which inherits from a built-in one) instead of assert here and in several other places that follow